Source/NSDockTile.m

```diff
@@ -45,6 +64,8 @@ - (oneway void) release
 {
   RELEASE(_contentView);
   RELEASE(_badgeLabel);
+  RELEASE(_appIconImage);
```

Comment:
As you did not retain this image a release is not needed. Or rather a retain would be needed on line 56.

Reply:
should be fixed too

Reply:
Could you please change the `retain` in line 56 to `RETAIN()`. Just to be consistent.